addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1530,8 +1526,8 @@ var SnippetsMenu = Widget.extend({
             kwargs: {
                 context: this.options.context,
             },
-        });
-        this.cacheSnippetTemplate[this.options.snippets] = this._defLoadSnippets;
+        }, { shadow: true });
```

Comment:
@detrouxdev Not sure, what's this change about?

Reply:
We switched to a silenced RPC because in the backend, an RPC that takes more than 3 seconds will trigger a blockUI showing an extra loader. This would sometimes happen when loading the snippets, and since there's already a loader on the edit button, we saw it as unnecessary.